src/ruby/ext/grpc/extconf.rb

```diff
@@ -62,7 +62,7 @@
 
 ENV['MACOSX_DEPLOYMENT_TARGET'] = '10.7'
 
-ENV['AR'] = RbConfig::CONFIG['AR'] + ' rcs'
+ENV['AR'] = RbConfig::CONFIG['AR'] unless grpc_config == 'lto'
```

Comment:
This has been moved to AROPTS in order to support protobuf's "AR" variable. I am not overloading the default basically.